docs/sdk-and-tools/notifier.md

```diff
@@ -0,0 +1,248 @@
+    # The pubsub channel used for publishing/subscribing
+    # Note: not required for running in the notifier mode
+    Channel = "pub-sub"
+
+    # The master name for failover client
+    MasterName = "mymaster"
+
+    # The sentinel url for failover client
+    SentinelUrl = "localhost:26379"
+
+    # The redis connection type. Options: | instance | sentinel |
+    # instance - it will try to connect to a single redis instance
+    # sentinel - it will try to connect to redis setup with master, slave and sentinel instances
+    ConnectionType = "sentinel"
+
+    # Time to live (in minutes) for redis lock entry
+    TTL = 30
+```
+
+For more details on notifier service setup, please follow the **Install** and **Launching**
+sections from [README](https://github.com/multiversx/mx-chain-notifier-go) in the repository.
+
+### Subscribers
+
+Currently there are 2 supported subscribing solutions:
+* RabbitMQ
+* Websocket
+
+The subscribing solution is selected based on a CLI parameter, please check
+[README](https://github.com/multiversx/mx-chain-notifier-go) from
+github repository for more info on the CLI parameters.
+
+In the notifier configuration directory (`cmd/notifier/config`), in `config.toml` there is
+a separate section `RabbitMQ`, which can be used to set up rabbitMQ connection url and
+exchanges. The exchanges will be created automatically (if they are not already created) on
+notifier service start.
+
+```toml
+[RabbitMQ]
+    # The url used to connect to a rabbitMQ server
+    # Note: not required for running in the notifier mode
+    Url = "amqp://guest:guest@localhost:5672"
+
+    # The exchange which holds all logs and events
+    [RabbitMQ.EventsExchange]
+        Name = "all_events"
+        Type = "fanout"
+
+    # The exchange which holds revert events
+    [RabbitMQ.RevertEventsExchange]
+        Name = "revert_events"
+        Type = "fanout"
+
+    ...
+```
+
+:::tip
+It is recommended to use the setup with RabbitMQ, if it is very important to avoid losing any event.
+:::
+
+## Events
+
+There are multiple event types:
+- Push Block event: when the block is commited, it contains logs and events
+- Revert Block event: when the block is reverted
+- Finalized Block event: when the block is finalized
+
+In RabbitMQ there is a separate exchange for each event type.
+In Websockets setup, there is a event type field in each message.
+
+The WS event is defined as following:
+
+| Field      | Description                                                                    |
+|------------|--------------------------------------------------------------------------------|
+| Type       | The type field defines the event type, it can be one of the followins: `all_events`, `revert_events`, `finalized_events`. `all_events` refers to all logs and events. |
+| Data       | Serialized data corresponding to the event type. |
+
+### Push Block Event
+
+Each time a block is commited on the chain, an event will be triggered with basic information
```

Comment:
committed

Reply:
search in the entire file and replace commited to committed

Reply:
changed :+1: